plugins/ros/src/ScenarioContext.tsx

```diff
@@ -145,7 +147,7 @@ const ScenarioProvider = ({
       setOriginalScenario(selectedScenario);
       setIsDrawerOpen(true);
     }
-  }, [riSc, scenarioIdFromParams, getRiScPath, navigate, isNewScenario]);
+  }, [riSc, scenarioIdFromParams, getRiScPath, navigate, searchParams]);
```

Comment:
```suggestion
  }, [riSc, scenarioIdFromParams, getRiScPath, navigate, scenarioWizardStep]);
```

Bør være så konkret som mulig

Reply:
ops, det ble feil 🙈 Siden `const scenarioWizardStep` blir definert inne i useEffecten.
Tenker du den bør bli definert utenfor, eller skal eg reverte tilbake til `searchParams` i dependency array?